src/components/Tabs.css

```diff
@@ -3,6 +3,6 @@
 	--background: #181924;
 }
 
-ion-tab-button{
+ion-tab-button {
```

Comment:
This will also affect all ion-tab-button elements. Would be better to use a class.